rust/gitxetcore/src/data/pointer_file.rs

```diff
@@ -178,6 +178,18 @@ impl PointerFile {
     }
 }
 
+pub fn is_xet_pointer_file(data: &[u8]) -> bool {
+    if data.len() >= 128 {
```

Comment:
Recommend not to use "128" although we know xet pointer file size is never going to exceed 128. Maybe switch to "POINTER_FILE_LIMIT" or any other constant var with comments.